rust/core-lib/src/tabs.rs

```diff
@@ -289,38 +308,36 @@ impl<W: Write + Send + 'static> Documents<W> {
         json!(view_id)
     }
 
-    fn do_close_view(&mut self, view_id: &ViewIdentifier) {
-        self.close_view(view_id);
+    fn close_view(&mut self, view_id: &ViewIdentifier) {
```

Comment:
The use of `do_` predates my involvement, and I don't have strong opinions. In `editor` the `do_` methods are generally the actual implementations?

It is not a pattern I'm using in new code.